src/sources/github_releases.rs

```diff
@@ -101,29 +101,16 @@ impl BackupSource<HttpFile> for GitHubReleasesSource {
         let target: GitHubRepoSourceKind = policy.from.as_str().parse()?;
 
         match target {
-          GitHubRepoSourceKind::User(u) if u.is_empty() => Err(errors::user(
-              &format!(
-                  "Your 'from' target '{}' is not a valid GitHub username.",
-                  policy.from.as_str()
-              ),
-              "Make sure you provide a valid GitHub username in the 'from' field of your policy.",
-          )),
-          GitHubRepoSourceKind::Org(org) if org.is_empty() => Err(errors::user(
-              &format!(
-                  "Your 'from' target '{}' is not a valid GitHub organization name.",
-                  policy.from.as_str()
-              ),
-              "Make sure you provide a valid GitHub organization name in the 'from' field of your policy.",
-          )),
-          GitHubRepoSourceKind::Repo(repo) if repo.is_empty() => Err(errors::user(
-              &format!(
-                  "Your 'from' target '{}' is not a fully qualified GitHub repository name.",
-                  policy.from.as_str()
-              ),
-              "Make sure you provide a fully qualified GitHub repository name in the 'from' field of your policy.",
-          )),
-          _ => Ok(()),
-      }
+            GitHubRepoSourceKind::Starred => Err(errors::user(
+                &format!(
+                    "Your 'from' target '{}' is not valid for 'kind' '{}'.",
```

Comment:
nit: The signature here is `errors::user(description, advice)` and usually we'll explain what went wrong (you cannot used 'starred' to backup releases) as the description and then provide advice like "Use 'from: user' or another supported source selector (users/<user>, orgs/<org>, repos/<repo>) instead." so that someone doesn't need to consult the docs to understand what to change (at least that's the dream).